test/1_stdlib/Optional.swift

```diff
@@ -1,226 +1,153 @@
-// RUN: %target-run-simple-swift | FileCheck %s
+// RUN: %target-run-simple-swift
 // REQUIRES: executable_test
 
+import StdlibUnittest
+import Swift
+
+let OptionalTests = TestSuite("Optional")
+
 protocol TestProtocol1 {}
 
 // Check that the generic parameter is called 'Memory'.
-extension Optional where Wrapped : TestProtocol1 {
+extension Optional where Wrapped: TestProtocol1 {
```

Comment:
In the standard library code, we put spaces on both sides of the colon when it denotes an "is-a" relationship: in generic parameter lists, in inheritance and protocol implementation lists etc.